project_euler/problem_01/sol7.py

```diff
@@ -0,0 +1,32 @@
+"""
```

Comment:
Please edit the filename to be `problem_07/sol7.py` so that it is in the appropriate directory.